ethpm_cli/parser.py

```diff
@@ -117,13 +120,32 @@ def add_alias_arg_to_parser(parser: argparse.ArgumentParser) -> None:
 #
 
 
-# todo: extend to pin & release a local manifest
 def release_cmd(args: argparse.Namespace) -> None:
     config = Config(args)
-    release_package(args.package_name, args.package_version, args.manifest_uri, config)
+
+    manifest_uri = args.manifest_uri
+
+    package_name = args.package_name
+    package_version = args.package_version
+
+    if args.manifest_path:
```

Comment:
This logic is great - I'm just not sure this is the best place for it to live. Maybe turn this into something like `pin_local_manifest(manifest_path)  -> package_name, package_version, manifest_uri`. And move it to `ethpm._utils.ipfs` should be a fine home.

There is manifest validation happening in web3. But, it's a good idea to throw in an extra validation check here with `from ethpm.validation.manifest import validate_manifest_against_schema`